c10/util/TypeCast.h

```diff
@@ -168,4 +168,21 @@ AT_FORALL_QINT_TYPES(DEFINE_UNCASTABLE)
 #undef DEFINE_UNCASTABLE
 #undef ERROR_UNSUPPORTED_CAST
 
+template <typename To, typename From>
```

Comment:
This was moved from somewhere else I guess?

Reply:
Yes, from Half.h